publications-2020.bib

```diff
@@ -185,16 +185,16 @@ @article{Motamarri2020
 }
 
 
-@article{10.1093/gji/ggaa039,
-    author = {K\"aufl, J S and Grayver, A V and Comeau, M J and Kuvshinov, A V and Becken, M and Kamm, J and Batmagnai, E and Demberel, S},
-    title = "{Magnetotelluric multiscale 3-D inversion reveals crustal and upper mantle structure beneath the Hangai and Gobi-Altai region in Mongolia}",
-    journal = {Geophysical Journal International},
-    year = {2020},
-    month = {01},
-    issn = {0956-540X},
-    doi = {10.1093/gji/ggaa039},
-    url = {https://doi.org/10.1093/gji/ggaa039},
-    note = {in press}
+@article{kaufl2020magnetotelluric,
+  title={Magnetotelluric multiscale 3-{D} inversion reveals crustal and upper mantle structure beneath the Hangai and {G}obi-{A}ltai region in {M}ongolia},
+  author={K{\"a}ufl, Johannes S and Grayver, Alexander V and Comeau, Matthew J and Kuvshinov, Alexey V and Becken, Michael and Kamm, Jochen and Batmagnai, Erdenechimeg and Demberel, Sodnomsambuu},
+  journal={Geophysical Journal International},
+  volume={221},
+  number={2},
+  pages={1002--1028},
+  year={2020},
+  publisher={Oxford University Press},
+  url = {https://doi.org/10.1093/gji/ggaa039}
```

Comment:
Could you please add the doi again? We aim to list both DOI and URL (even though they often refer to the same information) on the publications page.

Reply:
ok, done